config/crds/cluster_v1alpha1_machinedeployment.yaml

```diff
@@ -136,6 +136,24 @@ spec:
                         Use this to indicate what labels, annotations, name prefix,
                         etc., should be used when creating the Node.
                       type: object
+                    providerID:
```

Comment:
Why is there a providerID field on a MachineDeployment?

Reply:
PR: #565 

Removed the changes from here